.github/workflows/master.yml

```diff
@@ -276,8 +276,14 @@ jobs:
       - uses: ./.github/actions/pipdeptree
       - name: Run tests
         run: |
+          # Set current working directory to `tests/pyfunc` before running
+          # `test_inferred_data_path.py` so that `infer_code_paths` won't capture
+          # all modules under MLflow code repo.
+          # pushd tests/pyfunc && pytest test_inferred_data_path.py && popd
```

Comment:
can we use https://docs.pytest.org/en/latest/reference/reference.html#pytest.MonkeyPatch.chdir?

Reply:
Good idea!